privacyidea/lib/token.py

```diff
@@ -221,7 +221,8 @@ def _create_token_query(tokentype=None, realm=None, assigned=None, user=None,
             sql_query = sql_query.filter(Token.resolver == user.resolver)
         (uid, _rtype, _resolver) = user.get_user_identifiers()
         if uid:
-            sql_query = sql_query.filter(Token.user_id == str(uid))
+            uid = str(uid) if type(uid) == int else uid
```

Comment:
I find this harder to read than

~~~~python
if type(uid) == int:
    uid = str(uid)
~~~~

Especially since we do not need the else branch.
Is there any performance consideration or is it just style?

Reply:
Hm yeah, now that you mention it, I also find the two-liner easier to read :-) I don't think this makes any noticeable performance difference.

Reply:
Ok, i have to admit, i just wanted it to look fancy ;-)